modules/administration/pages/ansible-setup-control-node.adoc

```diff
@@ -3,7 +3,14 @@
 
 To set up an Ansible control node, execute the following steps from the {productname} {webui}.
 
-.Procedure: Setting up Ansible Control Node
+[NOTE]
+====
+The following procedure applies for {sle} 15 system only.
```

Comment:
We are not shipping ansible in any of our client tools anymore in 5.1. So we should adjust it accordingly. For every distribution, ansible needs to come from the distrubution.

For 15SP6 and SP7, it should be coming be coming from `Systems Management Module` which has dependency on `Python 3 Module` so both of these modules needs to be enabled if user wants to install ansible package on SP6/SP7 system in order to make them control node.

For SP4 and SP5, if user want to install new Ansible, they need to enable only Python 3 Module ` , this is what that bug was about where you were tagged.

We also need to add a note with newer ansible, systems with old python cannot be managed anymore.

Reply:
Thanks.  I added all this. I think here we shouldn't document Ansible version related limitations that detailed.  Focus here should be setting up a (new) Ansible control with MLM 5.1.